client/src/pages/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,97 @@
+import { useContext } from 'react';
+import FullCalendar, { EventClickArg, EventDropArg, EventHoveringArg } from '@fullcalendar/react'; // must go before plugins
+import dayGridPlugin from '@fullcalendar/daygrid'; // a plugin!
+import timeGridPlugin from '@fullcalendar/timegrid';
+import interactionPlugin, { EventResizeDoneArg } from '@fullcalendar/interaction';
+
+import { BoardContext } from '../../context/useBoardContext';
+import { ICard } from '../../interface';
+
+const Calendar = (): JSX.Element => {
+  const [board, setBoard] = useContext(BoardContext);
+
+  const handleClick = (info: EventClickArg) => {
+    // you need to render the card details popup here
+    alert(`${info.event.title} was clicked`);
+  };
```

Comment:
Not yet implemented?

Reply:
Yes I am blocked by issue #18 Detailed Card.